src/test/fixtures/editMode.ts

```diff
@@ -439,6 +439,28 @@ export const timezoneBlueprintResponse: BlueprintResponse = {
   description: mockBlueprintDescriptions['timezone'],
 };
 
+export const usersCreateBlueprintRequest: CreateBlueprintRequest = {
+  ...baseCreateBlueprintRequest,
+  name: mockBlueprintNames['users'],
+  description: mockBlueprintDescriptions['users'],
+  customizations: {
+    users: [
+      {
+        name: 'best',
+        password: '',
+        groups: [],
+        ssh_key: '',
```

Comment:
I don't think these should be here yet. In the User[] that we put together for the image-builder API, these are optional fields. Only the name should appear in the request for now.